app/views/content_items/get_involved.html.erb

```diff
@@ -94,31 +91,32 @@
     </section>
     <hr class="keyline">
     <section class="start-a-petition">
-      <div class="head-section">
+      <div class="head-section govuk-grid-column-one-third-from-desktop">
         <h3>Start a petition</h3>
       </div>
-      <div class="column">
+      <div class="column govuk-grid-column-two-thirds-from-desktop">
         <div class="content">
           <p>You can <a class="govuk-link" href="https://petition.parliament.uk/" rel="external">create a petition</a> to influence government and Parliament. If the petition gets at least 100,000 online signatures, it will be considered for debate in the House of Commons.</p>
         </div>
       </div>
     </section>
     <hr class="keyline">
     <section class="comment-follow">
-      <div class="head-section">
+      <div class="head-section govuk-grid-column-one-third-from-desktop">
         <h3>Follow a blog or social media channel</h3>
       </div>
-      <div class="column">
+      <div class="column govuk-grid-column-two-thirds-from-desktop">
         <div class="content">
           <p>For an instant way to interact with government departments, try their social media streams. These are listed under 'Follow us' on the department's home page. As well as access to blogs, audio, video and more, you can comment, debate and rate.</p>
           <p class="see-all"><a href="/government/organisations/" class="govuk-link">See all government departments</a></p>
           <p>Some active government blogs include:</p>
           <ul class="index-list">
             <li><a href="https://history.blog.gov.uk" class="govuk-link" rel="external">History of Government</a></li>
             <li><a href="https://quarterly.blog.gov.uk" class="govuk-link" rel="external">Civil Service Quarterly</a></li>
-            <li><a href="https://blogs.fcdo.gov.uk/" class="govuk-link" rel="external">FCDO bloggers</a></li>
+            <li><a href="http://blogs.fco.gov.uk" class="govuk-link" rel="external">FCO bloggers</a></li>
```

Comment:
This - and the change below - has slipped into the wrong commit.

Reply:
Only real way of solving this one is to include it in the original import of the html.erb as the change happened mid-development.